### DISCUSSION TITLE
Is that possible to run the  commercial project without running the abp login <username> command for any other developers ?

### DISCUSSION THREAD

**Discussion by ShifanilaSenthil** [abpframework/abp#18651] [Q&A] [answered]
Great product abp.io but we have some questions...

We have purchased a copy of abp commercial team edition, but we have a few questions that we would like answering.  if you could answer our question to give us a clearer picture of  abp commercial team edition?
1) Three of us (Developers) have used the purchased license/Credentials and started a base project using abp framework and created a repository with the same. Is that possible to clone that repository and run the project without running the abp login <username> command for any other developers ?
2) If so, How could that be possible ?
3) If not, How to make our rest of the developers to work on the same project with Abp commercial team edition ?

**Comment by ebicoglu** [comment, marked as answer]
all developers need to sign in via ABP CLI to be able to write code in the commercial version. for more information, contact at info@abpio

**Reply by ShifanilaSenthil** [reply]
Thanks for your Reply...
